CONTRIBUTING.md

```diff
@@ -175,51 +175,82 @@ Follow these steps to start contributing:
 5. Develop the features on your branch.
 
    As you work on the features, you should make sure that the test suite
-   passes:
+   passes. You should run the tests impacted by your changes like this:
+
+   ```bash
+   $ pytest tests/test_to_run.py
```

Comment:
How about the following, to make sure we don't get issues about `ERROR: file or directory not found: tests/test_to_run.py`?

```suggestion
   $ pytest tests/<TEST_TO_RUN>.py
```